lib/Workflow/Manager.php

```diff
@@ -163,7 +163,10 @@ public function registerWorkflow(string $workflowName, array $options = [])
         $this->workflows[$workflowName] = new WorkflowConfig($workflowName, $options);
 
         uasort($this->workflows, function (WorkflowConfig $a, WorkflowConfig $b) {
-            return $a->getPriority() < $b->getPriority();
+            if ($a->getPriority() == $b->getPriority()) {
+                return 0;
+            }
+            return ($a->getPriority() > $b->getPriority()) ? -1 : 1;
```

Comment:
That's a case for the spaceship operator:

```suggestion
            return $b->getPriority()) <=> $a->getPriority();
```